### PR TITLE
Cast 'n' seek

### LINKED ISSUE #286: Feature request: cast with seek
Feature request: a new `--seek` flag for `catt cast`, so that initialization of the cast, and setting the seek position to start playing from, can be done in a single command.

### PR DESCRIPTION
**Pull request by theychx**
fixes #286 :horse_racing: :horse_racing: :horse_racing:

### CODE REVIEW

**Comment by skorokithakis** [comment]
You're on fire! I will review when I'm at the PC and get back to you.

File: catt/cli.py [hunk 1]
```diff
@@ -160,6 +160,7 @@ def cli(ctx, device):
     help="YouTube-DL option. "
     "Should be passed as `-y option=value`, and can be specified multiple times (implies --force-default).",
 )
+@click.option("--seek", type=CATT_TIME, metavar="TIME", help="Start playback at specific timestamp.")
```
Comment (skorokithakis) [inline catt/cli.py:163]: Could we rename this to "seek-to"? I think that's a bit more explicit as a name, what do you think?

**Review by skorokithakis — APPROVED** [review]
Looks almost ready, I only have one comment!

File: catt/cli.py (hunk 1 above)
Comment (theychx) [reply catt/cli.py]: You're right. Fixed.

**Review by skorokithakis — APPROVED** [review]
Approved even more!

**Pull request merged**